packages/itwin/tree-widget/src/components/TreeWidgetUiItemsProvider.tsx

```diff
@@ -98,6 +98,12 @@ export class TreeWidgetUiItemsProvider implements UiItemsProvider {
         trees.push(...this._treeWidgetOptions.additionalTrees);
       }
 
+      if(this._treeWidgetOptions?.defaultTree && trees.length !== 0){
+        //Adding the defaultTree to first index
+        const defaultTreeId = this._treeWidgetOptions.defaultTree;
+        trees.unshift(trees.splice(trees.indexOf(trees.filter(tree => tree.id===defaultTreeId)[0]), 1)[0]);
```

Comment:
nice; but can we break this out from being one liner for better readability?

Reply:
Updated the code format based on comments.